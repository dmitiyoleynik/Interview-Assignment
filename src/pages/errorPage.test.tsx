import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import '@testing-library/jest-dom';

import ErrorPage from 'src/pages/errorPage';

test('ShipDetailsCard should match snapshot', () => {
  const component = renderer.create(
    <MemoryRouter>
      <ErrorPage />
    </MemoryRouter>,
  );
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
