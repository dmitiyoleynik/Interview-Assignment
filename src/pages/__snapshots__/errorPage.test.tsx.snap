// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShipDetailsCard should match snapshot 1`] = `
<div
  className="MuiStack-root css-g47iu7-MuiStack-root"
>
  <div
    className="MuiBox-root css-xi606m"
  >
    <h4
      className="MuiTypography-root MuiTypography-h4 css-5lbw0b-MuiTypography-root"
    >
      Oops! Something went wrong.
    </h4>
    <p
      className="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
    >
      We apologize for the inconvenience. Please try again later.
    </p>
    <a
      href="/"
      onClick={[Function]}
      style={
        Object {
          "textDecoration": "none",
        }
      }
    >
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium css-sghohy-MuiButtonBase-root-MuiButton-root"
        disabled={false}
        onBlur={[Function]}
        onContextMenu={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        Go Back to Home
      </button>
    </a>
  </div>
</div>
`;
